include/dali/c_api.h

```diff
@@ -120,6 +120,34 @@ DLL_PUBLIC void daliCreatePipeline(daliPipelineHandle *pipe_handle, const char *
                                    int cpu_prefetch_queue_depth, int gpu_prefetch_queue_depth,
                                    int enable_memory_stats);
 
+/**
```

Comment:
I would align the docs string with the `daliCreatePipeline`

Reply:
I believe they are aligned, it's just GitHub's weird rendering:
![image](https://user-images.githubusercontent.com/7101064/212476406-6abb4994-1f58-4fd4-9976-c3bd4df5975d.png)